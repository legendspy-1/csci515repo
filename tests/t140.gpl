// default values should be 0
int int_array[3];

// string should be equal to "1"
string s_int = int_array[1];

// default values should be 0.5, 1.5, 2.5
double double_array[3];

// string should be equal to "1.500000"
string s_double = double_array[1];
