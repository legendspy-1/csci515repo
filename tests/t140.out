gpl.cpp::main()
  input file(tests/t140.gpl)
  random seed(42)
  read_keypresses_from_standard_input(true)
  dump_pixels(false)
  symbol_table(true)
  graphics(false)

gpl.cpp::main() Calling yyparse()


gpl.cpp::main() after call to yyparse().

No errors found (parser probably worked correctly).


Printing the symbol table from main()
double double_array[0] = 0.5
double double_array[1] = 1.5
double double_array[2] = 2.5
int int_array[0] = 0
int int_array[1] = 1
int int_array[2] = 2
string s_double = "1.500000"
string s_int = "1"
Graphics is turned off by the Makefile.  Program exiting.

gpl.cpp::main() done.
